### DISCUSSION TITLE
Error in CPL File of PCB production

### DISCUSSION THREAD

**Discussion by RaW79** [revk/xESP32-Faikout#457] [Q&A] [answered]
Hi

just wanted to have a lool and uploaded your production files to JCLPCB but it stops with an error for the CPL file which seams to be the designations.csv in the git folder.

Guess the error is caused by the entry "REF**:1" in line 23
![grafik](https://github.com/user-attachments/assets/870001b8-ce23-48f8-87cf-8bfea76c12bf)

I was not able to figure out what component should be in there instead and kindly ask if anybody can support/correct the designations.csv file

Big thanks!

**Comment by revk** [comment]
Uh, you load the gerbers (zip), `bom.csv`, and `positions.csv`

Not `designators.csv`

**Reply by RaW79** [reply]
Without CPL JCLPCB offers no way to go on
![grafik](https://github.com/user-attachments/assets/7ccb80ec-19d9-4cd2-a15f-68f544a4e418)

**Reply by revk** [reply, marked as answer]
The CPL file is `positions.csv` - upload it!

**Reply by RaW79** [reply]
hello @revk are there also production files available for the faikin+ module?

**Reply by revk** [reply]
I make a dual interface one, that should be on here.

**Reply by RaW79** [reply]
Is it under developement or online? I´m looking around for the information about faikin+ module but could not find it anymore. Looking for a connection board to analyze S21 protocol. Thx!

**Reply by revk** [reply]
OK "Faikin+" has been the "next board" a bit.

I also made a dual input/output board for a while. But no, it is no longer in the repository. You can find in history, but it was just a test board.

There is no real Faikin+ as such now.

**Reply by RaW79** [reply]
Thanks, was not able to find so far but this sounds exactly what i was looking for. In the past I read sth. that you give it to special developers to make analysis on S21. So I´m looking for this dual port solution and because shipping abroad is more expensive than production I´m asking if this is still available.

**Reply by revk** [reply]
OK I have a few boards, I can post one if needed.

**Reply by RaW79** [reply]
Oh, would be great. Big Thanks in advance!

**Reply by revk** [reply]
I'll need address, and we should work out a price.

**Reply by RaW79** [reply]
Any suggestion for a communication channel for this? As far as I know github does not have a private chat/message function

**Reply by revk** [reply]
email support@aa.net.uk FAO Adrian
